web-app/src/main/webapp/index.html

```diff
@@ -84,7 +84,7 @@
     <div class="container u-full-width" id="dock">
     </div>
     <img id="report-button" src="images/report.svg" alt="button to report a new
-      incident">
+      incident" onclick="showReportForm()">
```

Comment:
nit: I know we were doing this for the starter tasks, but I believe the Right Way (tm) is to add it via JavaScript:

```
document.addEventListener('load', () => {
   document.getElementById('report-button').addEventListener('click', showReportForm);
});
```